src/screen/Council/CouncilScreen.tsx

```diff
@@ -69,5 +66,21 @@ export function CouncilScreen({navigation}: {navigation: NavigationProp<Dashboar
 }
 
 const styles = StyleSheet.create({
-  header: {justifyContent: 'space-between', flexDirection: 'row', padding: standardPadding * 2},
+  header: {
+    padding: standardPadding * 2,
+    flexDirection: 'row',
+    justifyContent: 'flex-end',
+  },
+  listHeader: {justifyContent: 'space-between', flexDirection: 'row', padding: standardPadding * 3},
+  content: {paddingVertical: standardPadding, paddingHorizontal: standardPadding * 2},
 });
+
+export function useCouncilMembers() {
+  const {api} = useContext(ChainApiContext);
+  return useQuery('members', async () => {
```

Comment:
Is there any other data we could have of members of other kind? How about `"council-members"` as queryKey?

Reply:
yes, `council-members` would be the same as the hook name also, I'll change it.